1Flow iOS doc update (#5322)
segmentio/segment-docs: src/connections/destinations/catalog/test-flow/index.md

---

> title: 1Flow Mobile Plugin Destination

Reviewer: ```suggestion
title: 1Flow Mobile Plugin Destination
id: 64dd07c1fed86b6866cd93f5
beta: true
hidden: true
```

---

> [1Flow](https://1flow.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a leading in-app user survey and messaging platform for Mobile app and SaaS businesses.

Reviewer: ```suggestion
[1Flow](https://1flow.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a leading in-app user survey and messaging platform for Mobile app and SaaS businesses.
```

---

> title: 1Flow (Actions) Destination

Reviewer 2: ```suggestion
title: 1Flow Mobile Plugin Destination
id: 64dd07c1fed86b6866cd93f5
```

Reviewer 3: Done. Thanks for suggestion

---

[1Flow](https://1flow.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is a leading in-app user survey and messaging platform for Mobile app and SaaS businesses.

Using 1Flow, you can reach users _in-the-moment_ while they are interacting with your website or application, to collect highly contextual user insights that help you improve your product offering and customer experience

> 

Reviewer 2: ```suggestion
The 1Flow Mobile Plugin Destination is open-source and available on GitHub. You can view these repositories here: 

- [iOS](github url){:target="_blank"} 
- [Android](github url){:target="_blank"}
```

Reviewer 3: Done

This destination is maintained by 1Flow. For any issues with the destination, [contact their Support team](mailto:support@1flow.app).

## Getting started

1. From the Segment web app, click **Catalog**, then click **1Flow**.
2. Find the Destinations Actions item in the left navigation, and click it.
3. Click **Configure 1Flow**.
4. Select an existing Source to connect to 1Flow (Actions).

> 5. Copy 1Flow project key and paste it into the Connection Settings in Segment.

Reviewer: Hi @rohantryskybox - where could a user find the 1Flow project key? Could we add a step or two here explaining how you could find the key in 1Flow?

Author: Done

> 1. From the Segment web app, click **Catalog**, then click **1Flow**.
> 2. Find the Destinations Actions item in the left navigation, and click it.
> 3. Click **Configure 1Flow**.
> 4. Select an existing Source to connect to 1Flow (Actions).
> 5. Go to 1flow.ai -> Settings -> Project Settings -> Copy 1Flow project key and paste it into the Connection Settings in Segment.

Reviewer 2: ```suggestion
1. From the Segment web app, click **Catalog**, then search for **1Flow Mobile Plugin**.
2. Click **Add Destination**.
4. Select an existing Source to connect to 1Flow Mobile Plugin.
5. Go to 1flow.ai -> Settings -> Project Settings, copy the 1Flow project key, and paste it into the Destination Settings in Segment.
6. Depending on the mobile source you’ve selected, include 1Flow's library by adding the following lines to your dependency configuration.
```

Reviewer 3: Done

> 

Reviewer 2: Add installation instructions for each library, see example documentation here: 

- Braze iOS installation: https://segment.com/docs/connections/destinations/catalog/braze/#ios
- Braze Android installation: https://segment.com/docs/connections/destinations/catalog/braze/#android

```suggestion
### iOS

1. Add the 1Flow Segment Pod to your `Podfile`:

    ```
    pod '1Flow-Segment'
    ```

(add other relevant instructions)

### Android

1. In the top-level project `build.gradle` add the following as a repository in *all projects > repositories*.

    ```js
    maven { url "http://appboy.github.io/1flow-android-sdk/sdk" }
    maven { url "http://appboy.github.io/1flow-segment-android/sdk" }
    ```

(add other relevant instructions)
```

Reviewer 3: Done. Also added setup instructions
